src/CustomSearch.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState, useRef, useEffect} from "react";
+
+function CustomSearch() {
+  const [adressForm, setAdressform] = React.useState('')
+  const [adressCoordinates, setAdresscoordinates] = React.useState('') 
+
+  function handleSubmit(event) {
+    event.preventDefault()
+    setAdressform(event.target.suggest.value)
+    
+    var myGeocoder = ymaps.geocode(event.target.suggest.value);
+    myGeocoder.then(
```

Comment:
Make `handleSubmit` `async` function and use `await` instead of `then` for promises.

```js
const func = async () => {
  await myPromise()

  // rest of code, and now the function looks liks synchronious, but it isn't
}
```

Reply:
Not done yet